### DISCUSSION TITLE
Placeholder variables

### DISCUSSION THREAD

**Discussion by NerdCowboy** [massCodeIO/massCode#65] [Ideas]
I've always used VSCode's snippets simply because their snippets allow you to use placeholder variables. It saves so much time just being able to tab through the snippet and change things, and I haven't been able to find anything that supports this.

Are there any plans to add this feature to massCode? 

I understand that it might not be straightforward to add to all platforms (ie Alfred/Raycast), but since you have direct integration with a VSCode extension, I'd presume it to be possible. Even with Alfred/Raycast, you could probably show a form with inputs for those variables that could generate the final snippet.

**Comment by antonreshetov** [comment]
@NerdCowboy I don't think it's easy to implement.  Inserting a snippet from an extension is essentially copy/paste, which is not the same as expanding a VS Code snippet where you can navigate through variables by tab. 

Overall I don't plan to think in that direction yet.

**Comment by danielo515** [comment]
What a shame.
What about offering a UI to fill the details, then generate the text?
